lib/sparrow/h2_worker.ex

```diff
@@ -1,6 +1,7 @@
 defmodule Sparrow.H2Worker do
```

Comment:
This module seems to be missing some potentially useful event emissions:
- connection lost
- connection attempt (fail + success)
- connection backoff / retry 
- worker init (since we have terminate, it's best to always track all symmetric events)